README.md

```diff
@@ -20,3 +20,15 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+## Answer 
+We know the best-case scenario for this code would be _O_(n) and that's if the array is already sorted when you put it in. It would just have to look at each element in the array and since it will be sorted the runtime will be based on the amount of elements there are.
+
+The worst-case scenario is infinite time or _O_(n!). The reason is because it will go on forever and there might not be an end to the sorting and even when it is sorted it might not be correct either. With this, it might never stop sorting because it has no way to see that it is sorted completely and is going to keep on going until it eventually sorts it. Which might be infinite. 
```

Comment:
How much work are you doing for each permutation? And how many permutations are there?

Reply:
But you're checking all possible permutations, aren't you? How can it run for an infinite amount of time then?